index.html

```diff
@@ -12,7 +12,17 @@
 </style>
 <div class="container">
   <sidebar-nav></sidebar-nav>
-  <reader-timeline></reader-timeline>
+  <div class="reader-container">
+    <div class="sort-container">
```

Comment:
Turn this into a custom element so we can keep it cleaner. Have it emit an event for when it changes and add a script on this page to set the `sort` attribute on the timeline element. The timeline element should listen on changes to its sort attribute.